activetable/templates/html/activetable_custom_report_format.html

```diff
@@ -0,0 +1,27 @@
+<div class="activetable_block"  data-score-type={{score_type}}>
+  <table id="activetable">
+    <colgroup>
+      {% for width in column_widths %}<col style="width: {{ width }}px;">{% endfor %}
+    </colgroup>
+    {% if headers_style %}
+      {{ headers_style|safe }}
+    {% else %}
+      <thead>
+        <tr class="active-table-custom-format-row" style="height: {{ head_height }}px;">
+          {% for cell in thead %}
+            <th class="active-table-custom-format-header" scope="col" style="border: 1px solid #c0c0c0; padding: 10px;">{{ cell }}</th>
+          {% endfor %}
+        </tr>
+      </thead>
+    {% endif %}
+    <tbody>
+      {% for row in tbody %}
+      <tr class="active-table-custom-format-row" style="height: {{ row.height }}px;">
+        {% for cell in row.cells %}
+          <td class="active-table-custom-format-cell"style="border: 1px solid #c0c0c0; padding: 10px;">{{cell.value}}</td>
```

Comment:
Maybe, all the inline css could be inside of the css file. @andrey-canon

Reply:
the custom_report_format method only can return this html as string, doesn't include another file or fragment  and the string can't include style labels

Reply:
Ok.